src/Domain/Stack/LiipTransformerStackExecutor.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Liip\ImagineBundle\Domain\Stack;
+
+use Imagine\Image\ImageInterface;
+use Imagine\Image\ImagineInterface;
+use Liip\ImagineBundle\Binary\MimeTypeGuesserInterface;
+use Liip\ImagineBundle\Domain\ImageReference;
+use Liip\ImagineBundle\Domain\ImageReferenceFile;
+use Liip\ImagineBundle\Domain\Imagine\Filter\FilterExecutor;
+use Liip\ImagineBundle\Domain\PostProcessor\PostProcessor;
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
+
+class LiipTransformerStackExecutor implements TransformerStackExecutor
```

Comment:
this is both the container for the filters and post processors and the executor for them.